### PR TITLE
Fixes to L96 preset PopovSandu

### PR DESCRIPTION
**Pull request by AndreyAPopov**

### CODE REVIEW

**Comment by Steven-Roberts** [comment]
Can you include a link to the paper?

**Comment by AndreyAPopov** [comment]
done

File: src/+otp/+lorenz96/+presets/PopovSandu.m [hunk 1]
```diff
@@ -1,56 +1,35 @@
...
+            fiveDays = 1;
+            omega    = 2 * pi/(fiveDays);
             
-            sixHours = 0.05;
-            day      = sixHours * 4;
-            numDays  = 5;
-            omega    = 2 * pi/(day * numDays);
-            
-            F = @(t) 8 + 4*cos(omega*(t + mod((1:40) - 1, q)/q)).';
+            F = @(t) 8 + 4*cos(omega*(t + mod((1:N) - 1, q)/q)).';
```
Comment (Steven-Roberts) [inline src/+otp/+lorenz96/+presets/PopovSandu.m:22]: This can be simplified to `F = @(t) 8 + 4*cos(2*pi*(t + (0:N-1)/q)).';`

Comment (Steven-Roberts) [reply src/+otp/+lorenz96/+presets/PopovSandu.m:22]: Or even `F = @(t) 8 + 4*cospi(2*(t + (0:N-1)/q)).';`

File: src/+otp/+lorenz96/+presets/PopovSandu.m [hunk 2]
```diff
@@ -1,56 +1,35 @@
...
+    
     methods
         function obj = PopovSandu(varargin)
             
             p = inputParser;
-            p.KeepUnmatched = true;
             addParameter(p, 'Size', 40, @isscalar);
+            addParameter(p, 'Partitions', 4, @isscalar);
 
             parse(p, varargin{:});
```
Comment (Steven-Roberts) [inline src/+otp/+lorenz96/+presets/PopovSandu.m:12]: Use `p.addParameter` and `p.parse`

File: src/+otp/+lorenz96/+presets/PopovSandu.m (hunk 1 above)
Comment (AndreyAPopov) [reply src/+otp/+lorenz96/+presets/PopovSandu.m]: I will make omega/2pi a parameter as that better reflects what it should be. Thanks for the tip about cospi that is very useful!

**Comment by Steven-Roberts** [comment]
I think it's ready to merge.

**Pull request merged**